roles/setup/tasks/main.yml

```diff
@@ -1,2 +1,108 @@
 ---
 # tasks file for setup
+
+- name: Check for CLUSTER_ALIAS input
+  fail:
+    msg: "You must supply '-e CLUSTER_ALIAS=<cluster_alias>'."
+  when:
+    - not CLUSTER_ALIAS is defined
```

Comment:
All of these `fail` checks could be reduced to a single fail using `with_items`, like this:

```
  - name: Ensure needed variables are set
    fail:
      msg: "Value {{ item }} is not set!"
    when: "{{ item }} is undefined"
    with_items:
    - one
    - two
    - three
```